tools/gen_ut_cmakelists.py

```diff
@@ -0,0 +1,133 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def parse_line(line):
+    name, os_, arch, timeout, run_type, launcher, dist_ut_port, run_serial, envs, conditions = line.strip(
+    ).split(",")
+
+    if envs[0:10] == "PYTHONPATH" or ";PYTHONPATH=" in envs:
```

Comment:
Split it first and then find match pattern.

Reply:
done